### PR TITLE
Feat/pypi

### PR DESCRIPTION
**Pull request by lheagy**
## Upload initial code to pypi
- automate deploy to pypi: https://pypi.python.org/pypi/gpgLabs so that you can now
```
pip install gpgLabs
```

## To deploy a new version of gpgLabs: 
- tests must be passing. 
- bump the version of the code using coda (https://github.com/aranzgeo/coda)
```
coda patch  # or coda minor / major (depending on the scale of the changes) 
```
- generate a release on github: https://github.com/geoscixyz/gpgLabs/releases

Then the code will automatically deploy to pypi from travis. If for some reason you need to deploy code that is failing or the auto-deploy does not work, @thast has permission on pypi and knows how to manually upload.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,23 +2,36 @@ language: python
...
+  - pip install twine;
+  - cd gpgLabs; python setup.py install; cd ..
 
 # Run test
 script:
   - nosetests $TEST_DIR -v -s
 
+# deploy to pypi
+after_success:
+  - if [ "$TRAVIS_TAG" ]; then
```
Comment (lheagy) [inline .travis.yml]: a new version of the code needs to be committed and a github release completed for the code to auto-deploy
```
coda patch
```
(https://github.com/aranzgeo/coda) 
cc @thast

File: README.rst [hunk 2]
```diff
@@ -1,8 +1,14 @@
+.. _gpgLabs:
+
 gpgLabs
 =======
 
 Tutorials and examples for applied geophysics!
 
+.. image:: https://img.shields.io/pypi/v/gpgLabs.svg
```
Comment (lheagy) [inline README.rst]: added the pypi badge to the readme

**Comment by thast** [comment]
@lheagy : is this PR still active?

**Comment by lheagy** [comment]
ah, thanks for the reminder. This is currently out-of-date. 

todo's:
- [ ] update branch to be inline with most recent master
- [ ] update credentials to be consistent with PyPi upgrades

**Comment by lheagy** [comment]
closing this in favour of #68

**Pull request closed without merging**